ipc_plugin/ipc_mod.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3.5m
```

Comment:
You do not need hashbangs on modules that are not supposed to run independently.